melodic/distribution.yaml

```diff
@@ -3878,6 +3878,21 @@ repositories:
       url: https://github.com/jackal/jackal.git
       version: kinetic-devel
     status: maintained
+  jackal_cartographer_navigation:
+    doc:
+      type: git
+      url: https://github.com/jackal/jackal_cartographer_navigation.git
+      version: melodic-devel
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/clearpath-gbp/jackal_cartographer_navigation-release.git
+      version: 0.0.1-1
+    source:
+      type: git
+      url: https://github.com/jackal/jackal_cartographer_navigation.git
```

Comment:
Can you please add a LICENSE file to the source repository?  Thanks.

Reply:
Added to all, shall I re-bloom?

Reply:
Not really necessary; it won't effect the binary distribution.  Thanks for iterating.